### DISCUSSION TITLE
osu!lazer sound not working

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#26119] [Q&A] [answered]
### Type

Other

### Bug description

The correct output device is selected, but there is no sound.

### Screenshots or videos

![image](https://github.com/ppy/osu/assets/135897547/c5744e87-cb34-4115-a0f5-833490e3c3c5)
![image](https://github.com/ppy/osu/assets/135897547/04018451-706e-42c1-a4b5-6deac0083103)

### Version

2023.1224.0-lazer OS: ArchLinux

### Logs

[compressed-logs.zip](https://github.com/ppy/osu/files/13765059/compressed-logs.zip)

**Comment by gepbird** [comment]
I have this issue too on NixOS with 1224 and 1229 release client.

When obs (or any application that is trying to use that audio output device) is running, starting osu! results in the aforementioned error. Closing obs and restarting osu! fixes this problem.
A few days ago I randomly got this error multiple times on startup (probably some audio was playing when I started the game) and changing the output device to something else then back to the original fixed it.

This issue happens in reverse: if I first start osu! first then other applications, I can hear the game well, but can't see the desktop audio in obs, videos on Youtube, Discord or with mpv refuse to play (or sometimes play without audio).

@snarexdev as a workaround you can try stopping everything that produces sounds and start osu! after that. If you still get this issue, try changing the output device in osu to something else then back to the original.

**Comment by ghost** [comment, marked as answer]
Most likely, the problem was an incorrectly installed audio server, I installed elementaryOS 7.1 and osu!lazer started out of the box and the sound works fine. Topic closed

**Comment by gepbird** [comment]
@snarexdev https://github.com/Quaver/Quaver/issues/933#issuecomment-509429310
TLDR: you need to install the pulseaudio-alsa package

Unfortunately I have a pipewire config makes the buffer bigger as a performance workaround, and using this config with alsa enabled I get sound stutters in the game. However without this pipewire config, the BASS 46 error is gone and sound works.

**Comment by rszyma** [comment]
I get that error sometimes too. What helps is exiting osu, closing all apps that might play audio, wait for 10 seconds, and then run osu again.
